stages/org.osbuild.oscap.remediation

```diff
@@ -61,63 +63,76 @@ SCHEMA = """
       },
-      "arf_results": {
+      "xccdf_results": {
         "type": "string",
-        "description": "Filename for saving the arf result file"
+        "description": "Filename for storing the XCCDF Results file"
       },
       "html_report": {
         "type": "string",
-        "description": "Filename for saving the html report",
-        "default": "eval_remediate_report.html"
+        "description": "Filename for saving the final HTML report"
       },
       "verbose_log": {
         "type": "string",
         "description": "Filename for verbose error messages"
       },
       "verbose_level": {
         "type": "string",
-        "enum": ["DEVEL", "INFO", "ERROR", "WARNING"],
+        "enum": ["DEVEL", "INFO", "WARNING", "ERROR"],
         "description": "The verbosity level for the log messages"
+      },
+      "compress_results": {
+        "type": "boolean",
+        "default": false,
+        "description": "Compress ARF and XCCDF results file(s) with xz"
       }
     }
   }
 }
 """
 
-XML_RESULTS = "eval_remediate_results.xml"
-HTML_REPORT = "eval_remediate_report.html"
+DATA_DIR = "/root"
+XCCDF_RESULTS = "oscap_eval_xccdf_results.xml"
+REMEDIATION_SCRIPT = "oscap_remediation.bash"
 
 
 def main(tree, options):
     # required vars
     config = options["config"]
-    profile = config["profile_id"]
+    profile_id = config["profile_id"]
     datastream = config["datastream"]
     # optional vars
     xccdf_id = config.get("xccdf_id")
-    data_dir = options.get("data_dir")
+    data_dir = options.get("data_dir", DATA_DIR)
     ds_id = config.get("datastream_id")
     tailoring = config.get("tailoring")
-    xml_results = config.get("xml_results", XML_RESULTS)
-    html_report = config.get("html_report", HTML_REPORT)
+    arf_results = config.get("arf_results")
+    xccdf_results = config.get("xccdf_results", XCCDF_RESULTS)
+    html_report = config.get("html_report", "report.html")
+    verbose_log = config.get("verbose_log", "log")
+    verbose_level = config.get("verbose_level", "INFO" if verbose_log is not None else None)
+    compress_results = config.get("compress_results", False)
 
-    # run openscap in chroot on the image tree
+    data_dir = data_dir.lstrip('/')
+    os.makedirs(f"{tree}/{data_dir}", exist_ok=True)
+
+    # run openscap in chroot on the image tree (scan)
     cmd = [
         "/usr/sbin/chroot", tree,
-        "/usr/bin/oscap", "xccdf", "eval",
-        "--remediate", "--profile", profile
+        "/usr/bin/oscap",
+        "xccdf", "eval"
     ]
 
-    if data_dir is not None:
-        os.makedirs(f"{tree}/{data_dir.lstrip('/')}", exist_ok=True)
-        # run with command in chroot so full path ok
-        cmd.extend(["--results", f"{data_dir}/{xml_results}"])
-        cmd.extend(["--report", f"{data_dir}/{html_report}"])
+    if verbose_level is not None:
+        cmd.extend(["--verbose", verbose_level])
+
+    if verbose_log is not None:
```

Comment:
`verbose_log` can't ever be `None`, since the default value is `"log"` if not provided to the stage in its options.

Reply:
Fixed.